Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
I love Pillar and I dont like the verbosity of Latex, it reminds me too much of html/xml. So thank you for Pillar it definitely made me a more active contributor. Pillar has a long way to go to compete with Latex feature wise, but I think its on the right road.